digdag-docs/src/command_reference.rst

```diff
@@ -367,13 +367,16 @@ In the config file, following parameters are available
 * database.maximumPoolSize (integer, default: available CPU cores * 32)
 * database.leakDetectionThreshold (HikariCP leakDetectionThreshold milliseconds in integer. default: 0. To enable, set to >= 2000.)
 * database.migrate (enable DB migration. default: true)
-* archive.type (type of project archiving, "db" or "s3". default: "db")
+* archive.type (type of project archiving, "db", "s3" or "gcs". default: "db")
 * archive.s3.endpoint (string. default: "s3.amazonaws.com")
 * archive.s3.bucket (string)
 * archive.s3.path (string)
 * archive.s3.credentials.access-key-id (string. default: instance profile)
 * archive.s3.credentials.secret-access-key (string. default: instance profile)
 * archive.s3.path-style-access (boolean. default: false)
+* archive.gcs.bucket (string)
+* archive.gcs.credentials.json (string)
+* archive.gcs.credentials.json.content (string)
```

Comment:
Could you explain this two key setting in detail. I don't what should be stored.

Reply:
What do you think to write the configuration example like the following?
http://docs.digdag.io/command_reference.html#secret-encryption-key

Reply:
`gcs.credentials.json` indicates the json path where the GCP credentials are written.
`gcs.credentials.json.content` directly writes GCP credentials.

May I write after (string)?

Reply:
> gcs.credentials.json indicates the json path where the GCP credentials are written.

Just idea,  _gcs.credentials.json.path_ may be better.
When user write both _gcs.credentials.json_ and _gcs.credentials.json.content_ , which is selected or becoming error ? Please describe about it.

> May I write after (string)?

Basically good if it is not too much.

Reply:
When user write both gcs.credentials.json and gcs.credentials.json.content, will be selected `gcs.credentials.json`.